cmds/cmd.c

```diff
@@ -196,7 +197,17 @@ void cmd_prompt(void)
 
 		/* Regular characters */
 		if (c) {
-			if (c == '\r') {
+			if ((c == '\r') || (c == '\n')) {
+				/* handle crlf line ending */
+				if ((c == '\r') && (newline == 0)) {
```

Comment:
`newline == 0` is redundant - we always end up with `newline` = 1 anyway
```suggestion
				if (c == '\r') {
```

Reply:
Yeah, maybe that's better, changed.